storage/boltdb/net-state.go

```diff
@@ -0,0 +1,82 @@
+package boltdb
+
+import (
+	"encoding/json"
+	"errors"
+	"log"
+
+	"github.com/boltdb/bolt"
+)
+
+type File struct {
+	Path  string `json:"path"`
+	Value string `json:"value"`
+}
+
+var (
+	ErrCreatingFileBucket = errors.New("error creating file bucket")
+	ErrFileNotFound       = errors.New("error file not found")
+	ErrIterKeys           = errors.New("error unable to iterate through bucket keys")
+	ErrDeletingFile       = errors.New("error unable to delete file key")
+)
+
+func (client *Client) Put(file File) error {
+	return client.db.Update(func(tx *bolt.Tx) error {
+		b, err := tx.CreateBucketIfNotExists([]byte("files"))
+		if err != nil {
+			return ErrCreatingFileBucket
+		}
+
+		fileKey := []byte(file.Path)
+
+		fileBytes, err := json.Marshal(file)
```

Comment:
could just use file.Value for fileBytes, since fileKey is already stored in the database elsehow. that way you aren't storing the file.Path twice